force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -94,17 +115,31 @@ public with sharing class DummyJSONCallout {
 	
 	public static void postCreateDummyJSONUser(String contactId) {
 		// Create HTTP request to send.
-
+        HttpRequest request = new HttpRequest();
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
-
+        request.setEndpoint('callout:DummyJsonUser' + 'add');
 		// Set the HTTP method to POST.
-
+        request.setMethod('POST');
 		// Set the body using generateDummyJsonUserPayload method.
-
+        request.setBody(generateDummyJsonUserPayload(contactId));
 		// Send the HTTP request and get the response.
-
+       Http http = new Http();
+	   HttpResponse response = http.send(request);
 		// If the HTTP response code is successful, update the contact.
+		if (response.getStatusCode() >= 200 || response.getStatusCode() <= 299) {
```

Comment:
```suggestion
		if (response.getStatusCode() >= 200 && response.getStatusCode() <= 299) {
```
This should be AND, not OR. You want the response to be between 200 and 299 to be a success. Other response codes indicate some sort of failure.